lib/python/qmk/tests/test_cli_commands.py

```diff
@@ -84,3 +110,39 @@ def test_list_keymaps_no_keyboard_found():
     result = check_subcommand('list-keymaps', '-kb', 'asdfghjkl')
     assert result.returncode == 0
```

Comment:
```suggestion
    check_returncode(result, 1)
```

For the other `list-keymap` checks the `assert result.returncode == 0` should be switched to `check_returncode(result)`.